### PR TITLE
Gallery images

### PR DESCRIPTION
**Pull request by karenm08**
#57 #26 
fixed contact page validation

### CODE REVIEW

**Review by Wayl123 — APPROVED** [review]
Added contact.js back weeeeee.

File: scss/gallery/_content.scss [hunk 1]
```diff
@@ -1,3 +1,9 @@
+h1 {
+    font-size: 40px;
+    color: #444444;
+    font-family: 'Cormorant', serif;
```
Comment (stevenlai4) [inline scss/gallery/_content.scss:4]: I put this line in the body scss as the global font already

File: scss/contact/contact.scss [hunk 2]
```diff
@@ -10,49 +10,61 @@
...
     padding: 2%;
-    width: 80%;
+    width:90%;
     background-color: #dfd3c3;
-    border-radius: 5px;
+    border-radius: 10px;
     margin-bottom: 5%;
     margin-top: 5%;
-    color: #000;
+    font-family: 'Cormorant', serif;
```
Comment (stevenlai4) [inline scss/contact/contact.scss:20]: This is in global body scss as well

**Review by stevenlai4 — CHANGES_REQUESTED** [review]
I put comments below

**Review by Wayl123 — CHANGES_REQUESTED** [review]
Yeah I also notice the error steven is talking about.

**Review by Wayl123 — APPROVED** [review]
The error I notice have been fixed -w-b.

**Pull request merged**